barter-data/src/exchange/coinbase/mod.rs

```diff
@@ -32,10 +34,13 @@ pub mod subscription;
 /// Public trade types for [`Coinbase`].
 pub mod trade;
 
+/// Public book types for [`Coinbase`].
+pub mod book;
+
 /// [`Coinbase`] server base url.
 ///
 /// See docs: <https://docs.cloud.coinbase.com/exchange/docs/websocket-overview>
-pub const BASE_URL_COINBASE: &str = "wss://ws-feed.execution.coinbase.com";
+pub const BASE_URL_COINBASE: &str = "wss://ws-feed.exchange.coinbase.com";
```

Comment:
Would you mind filling me in on the reason for this change? Has some API version been deprecated?